easybuild/easyblocks/g/gromacs.py

```diff
@@ -201,14 +238,48 @@ def configure_step(self):
                 run_cmd(plumed_cmd, log_all=True, simple=True)
 
         else:
+            if '-DGMX_MPI=ON' in self.cfg['configopts']:
+                if self.cfg.get('mpi_numprocs', 0) == 0:
+                    self.log.info("No number of test MPI tasks specified -- using default: %s",
+                                  self.cfg['parallel'])
+                    self.cfg['mpi_numprocs'] = self.cfg['parallel']
+
+                elif self.cfg.get('mpi_numprocs') > self.cfg['parallel']:
+                    self.log.warning("Number of test MPI tasks (%s) is greater than value for 'parallel': %s",
+                                     self.cfg.get('mpi_numprocs'), self.cfg['parallel'])
+
+                mpiexec = which(self.cfg.get('mpiexec'))
+                if mpiexec:
```

Comment:
You should be a bit more careful here, usign `which(None)` would end in tears:

```python
mpiexec = self.cfg.get('mpiexec')
if mpiexec:
    mpiexec = which(mpiexec)
    if mpi_exec:
        ...
    elif self.cfg['runtest']:
        raise EasyBuildError("'%s' not found in $PATH", mpiexec')
elif self.cfg['runtest']:
    raise EasyBuildError("No value found for 'mpiexec'")

Reply:
Done